README.rst

```diff
@@ -61,14 +60,40 @@ For more information, see `Getting Started`_.
 Basic usage
 ^^^^^^^^^^^
 
-This code shows how to import PyHPS Data Transfer and use some basic capabilities:
+The following sections show how to import PyHPS Data Transfer and use some basic capabilities:
+
+Requesting access token
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The client library requires an access token to connect to the HPS Data Transfer service.
+
+.. code:: python
+
+    from ansys.hps.data_transfer.client.authenticate import authenticate
+
+    auth_url = "https://localhost:8443/hps/auth/realms/rep"
+
+    token = authenticate(username="repadmin", password="repadmin", verify=False, url=auth_url)
+    token = token.get("access_token", None)
+
+Connect to data transfer service
```

Comment:
Should client be part of the sentence tile as it is used in the first sentence? We want to use terms consistently.